### DISCUSSION TITLE
Add a new Dynamic Tag to Count Terms

### DISCUSSION THREAD

**Discussion by Michaelgimii** [elementor/elementor#22842] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi

A new dynamic tag to display term count would be great. Especially when taxonomy support is added for the loop builder, this dynamic tag will be very useful.

### Describe the solution you'd like

New dynamic tag for terms : Count Term

**Comment by nicholaszein** [comment]
This Feature Request was mentioned in the following Issues and Discussions:

- #23954

**Reply by caspervoogt** [reply]
I went to https://github.com/orgs/elementor/discussions/23954 to upvote it, because it describes counting posts (not terms) and that differs from this issue. However, https://github.com/orgs/elementor/discussions/23954 is locked (don't know why). 

It has baffled me for years why Elementor's Posts, Archive Posts and now Loop Grid elements don't come with a 'results count' option of some kind. It is the kind of thing I use a lot on sites, and I doubt I'm the only one. I know it is possible with third-party addons but I feel like it should be part of Elementor.
